src/v3/src/util/getSocialIdpButtonIcon.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import adobeIconSvg from '../img/socialButtonIcons/adobe.svg';
+import amazonIconSvg from '../img/socialButtonIcons/amazon.svg';
+import appleIconSvg from '../img/socialButtonIcons/apple.svg';
+import discordIconSvg from '../img/socialButtonIcons/discord.svg';
+import facebookIconSvg from '../img/socialButtonIcons/facebook.svg';
+import githubIconSvg from '../img/socialButtonIcons/github.svg';
+import gitlabIconSvg from '../img/socialButtonIcons/gitlab.svg';
+import googleIconSvg from '../img/socialButtonIcons/google.svg';
+import lineIconSvg from '../img/socialButtonIcons/line.svg';
+import linkedinIconSvg from '../img/socialButtonIcons/linkedin.svg';
+import msIconSvg from '../img/socialButtonIcons/ms.svg';
+import oktaIconSvg from '../img/socialButtonIcons/okta.svg';
+import orcidIconSvg from '../img/socialButtonIcons/orcid.svg';
+import paypalIconSvg from '../img/socialButtonIcons/paypal.svg';
+import quickbooksIconSvg from '../img/socialButtonIcons/quickbooks.svg';
+import salesforceIconSvg from '../img/socialButtonIcons/salesforce.svg';
+import spotifyIconSvg from '../img/socialButtonIcons/spotify.svg';
+import xeroIconSvg from '../img/socialButtonIcons/xero.svg';
+import yahooIconSvg from '../img/socialButtonIcons/yahoo.svg';
+import yahooJapanIconSvg from '../img/socialButtonIcons/yahoo-japan.svg';
+
+export const getSocialIdpButtonIcon = (idpType: string) : string => {
```

Comment:
Instead of a function, would it make more since to just export the constant `idpIconMapping`, then use that constant in the formUtils?

Reply:
yeah that does make more sense, changing that now